content/admin/Administration Panel/Roles and Security Groups.adoc

```diff
@@ -247,7 +247,8 @@ For more help and information on setting up and configuring roles and security g
 please see 
 
 * link:/user/security-suite-groups/#_a_typical_hierarchy_setup[Example of a Typical Setup]
-* https://www.youtube.com/watch?v=yJ-BzM3GTgA[Introduction Video^]
+* https://www.youtube.com/watch?v=yJ-BzM3GTgA[Introduction Video]
+
 
 
 
```

Comment:
Not related to any change you made now, just something I noticed: it would be nice to elevate this sentence to a "tip". It saves a lot of time, and people often overloook it.

Reply:
> Note that you can change the setting for an entire column by clicking on the column heading.

Reply:
It appears further down in this file.